src/translations/strings.yaml

```diff
@@ -928,6 +928,19 @@ devices:
     value: My devices
     comment: Title for the menu bar when viewing the devices or device limit pages
 
+  privacyTitle:
+    value: Privacy features
+    comment: Title label for the privacy features help sheet
+  privacyHeader:
+    value: What do privacy features do for me?
```

Comment:
"What do…do" doesn't sound great?

"What can…do" looks like a minimal change here.

Reply:
@traceyvantyghem

Reply:
Yes, I think this is a good suggestion, especially because "do" implies they do it by default, but "can" is more aligned with the fact that users can turn things off or on here.

Anyway, I agree, I think we should update to "What can privacy features do for me?"